js/components/ControlButton.tsx

```diff
@@ -20,6 +20,6 @@ export interface ControlButtonProps {
  */
 export const ControlButton = ({ icon, title, onClick, ...props }: ControlButtonProps) => (
     <button onClick={onClick} title={title} {...props} >
-        <i class={`fas fa-${icon}`}></i>
+        <i className={`fas fa-${icon}`}></i>
```

Comment:
What's the difference between `class` and `className`?

Reply:
`className` is the React/Preact keyword, to avoid collision with the `class` Javascript keyword, although in practice this doesn't actually seem to be an issue.